source/WindowScroller/WindowScroller.example.js

```diff
@@ -9,19 +9,27 @@ import AutoSizer from '../AutoSizer'
 import shallowCompare from 'react-addons-shallow-compare'
 import styles from './WindowScroller.example.css'
 
-export default class AutoSizerExample extends Component {
+export default class WindowScrollerExample extends Component {
   static contextTypes = {
-    list: PropTypes.instanceOf(Immutable.List).isRequired
+    list: PropTypes.instanceOf(Immutable.List).isRequired,
+    customElement: PropTypes.object,
```

Comment:
In an earlier commit, I had `PropTypes.instanceOf(Element)`, but the linter complained about that, and I realized that it’s probably for good reason: `Element` won’t be defined on servers, and this is a static property. So, not sure if there’s something better than `object` here? I guess we could make a custom validator function which, when invoked, looks at `window.Element` but gracefully does nothing on server-side render?